src/NServiceBus.RabbitMQ/Connection/RabbitMqConnectionFactory.cs

```diff
@@ -44,10 +44,9 @@ public RabbitMqConnectionFactory(ConnectionConfiguration connectionConfiguration
             };
         }
 
-        public virtual IConnection CreateConnection(string purpose)
+        public virtual IConnection CreateConnection(string connectionName)
```

Comment:
> they're both internal classes

Well, yes, that changes things :laughing: 

OK, ignore all of the above. Sorry for derailing. :grin: